packages/flutter/lib/src/rendering/mouse_cursor.dart

```diff
@@ -416,45 +416,425 @@ class SystemMouseCursors {
   /// Any cursor other than [none] or [uncontrolled] unhides the cursor.
   static const SystemMouseCursor none = SystemMouseCursor._(kind: 'none');
 
+
+  //// STATUS ////
+
   /// The platform-dependent basic cursor.
   ///
   /// Typically the shape of an arrow.
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_DEFAULT, TYPE_ARROW
+  ///  * Web: default
+  ///  * macOS: arrowCursor
   static const SystemMouseCursor basic = SystemMouseCursor._(kind: 'basic');
 
-  /// A cursor that indicates a user interface element that is clickable, such as a hyperlink.
+  /// A cursor that emphasizes an element being clickable, such as a hyperlink.
   ///
   /// Typically the shape of a pointing hand.
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_HAND
+  ///  * Web: pointer
+  ///  * macOS: pointingHandCursor
   static const SystemMouseCursor click = SystemMouseCursor._(kind: 'click');
 
-  /// A cursor that indicates selectable text.
+  /// A cursor indicating an operation that will not be carried out.
+  ///
+  /// Typically the shape of a circle with a diagonal line. May fall back to
+  /// [noDrop].
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_NO_DROP
+  ///  * Web: not-allowed
+  ///  * macOS: operationNotAllowedCursor
+  ///
+  /// See also:
+  ///
+  ///  * [noDrop], which indicates somewhere that the current item may not be
+  ///    dropped.
+  static const SystemMouseCursor forbidden = SystemMouseCursor._(kind: 'forbidden');
+
+  /// A cursor indicating the status that the program is busy and therefore
+  /// can not be interacted with.
+  ///
+  /// Typically the shape of an hourglass or a watch.
+  ///
+  /// Corresponds to:
+  ///
+  ///  * Android: TYPE_WAIT
+  ///  * Web: wait
+  ///
```

Comment:
Is there no mac equivalent? Like the spinning beach ball? 😜

Reply:
I probably should doc this, but mac decides when to show the spinning beach ball by itself (when the application does not respond a blocked callback for a while), and does not expose a way to manually trigger it.